easybuild/easyconfigs/d/DAAL/DAAL-2017.1.132.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = "IntelBase"
+name = 'DAAL'
+version = '2017.1.132'
+
+homepage = 'http://software.intel.com/en-us/intel-daal'
+description = """Boost machine learning and big data analytics performance 
+ with easy-to-use library.
+ Features highly tuned functions for deep learning, classical 
+ machine learning, and data analytics performance across spectrum 
+ of Intel architecture devices.
+ Optimizes data ingestion together with algorithmic computation for 
+ highest analytics throughput.
+ Includes Python*, C++, and Java* APIs and connectors to popular data 
+ sources including Spark* and Hadoop*.
+ Free and open source community-supported versions are available, as 
+ well as paid versions that include premium support."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_%(name)s_%(version)s.tgz']
+
+dontcreateinstalldir = 'True'
+
+license_file = HOME + '/licenses/intel/license.lic'
+
+modextrapaths = {
+    'LIBRARY_PATH': ['compilers_and_libraries_%(version)/linux/daal/lib/intel64_lin',
+                     'compilers_and_libraries_%(version)/linux/daal/../tbb/lib/intel64_lin/gcc4.4'],
```

Comment:
Can you collapse this to `compilers_and_libraries_%(version)/linux/tbb/lib/intel64_lin/gcc4.4`? Same below?

Can we avoid hardcoding the `gcc4.4` part? Since that will be easily overlooked when updating to a newer version of DAAL...

Same below for `LD_LIBRARY_PATH`

Reply:
Intel seems to ship libtbb library for gcc4.1, 4.4, 4.7 but installation is using gcc4.4

```
/nfs/grid/software/RHEL7/easybuild/software/Core/daal/2017.1.132/compilers_and_libraries_2017.1.132/linux/tbb/lib/intel64_lin/gcc4.1:
total 5222
-rwxr-xr-x 1 root root      26 Apr 25  2017 libtbbmalloc.so
-rwxr-xr-x 1 root root  894886 Apr 25  2017 libtbbmalloc.so.2
-rwxr-xr-x 1 root root      20 Apr 25  2017 libtbb.so
-rwxr-xr-x 1 root root 2498855 Apr 25  2017 libtbb.so.2

/nfs/grid/software/RHEL7/easybuild/software/Core/daal/2017.1.132/compilers_and_libraries_2017.1.132/linux/tbb/lib/intel64_lin/gcc4.4:
total 4422
-rwxr-xr-x 1 root root      26 Apr 25  2017 libtbbmalloc.so
-rwxr-xr-x 1 root root  781349 Apr 25  2017 libtbbmalloc.so.2
-rwxr-xr-x 1 root root      20 Apr 25  2017 libtbb.so
-rwxr-xr-x 1 root root 2061389 Apr 25  2017 libtbb.so.2

/nfs/grid/software/RHEL7/easybuild/software/Core/daal/2017.1.132/compilers_and_libraries_2017.1.132/linux/tbb/lib/intel64_lin/gcc4.7:
total 4502
-rwxr-xr-x 1 root root      26 Apr 25  2017 libtbbmalloc.so
-rwxr-xr-x 1 root root  780272 Apr 25  2017 libtbbmalloc.so.2
-rwxr-xr-x 1 root root      20 Apr 25  2017 libtbb.so
-rwxr-xr-x 1 root root 2106456 Apr 25  2017 libtbb.so.2

```

Reply:
So why use the `gcc4.4`, and not the `gcc4.7`? What determines which library you pick?

Another option is that we include a self-built `tbb` (which provides the same libraries) as dependency to DAAL (we already have easyconfigs for `tbb` built from source)...